src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -168,6 +167,7 @@ class CodyAutocompleteManager {
     val autocompleteRequest =
         triggerAutocompleteAsync(
             project, editor, offset, textDocument, triggerKind, cancellationToken)
+    if (!isAsync) autocompleteRequest.join()
```

Comment:
This code along with `disableAsyncAutocomplete` was added only for testing purposes. 

IMO this looks very bad, but I don't have any other idea on how to ensure that the CompletableFuture completes within this scope.

Reply:
Any ideas?

Or we can just merge that and I'll fix that later.

Reply:
I have one idea but I did not very carefully verified all corner cases.
Looks like `myFixture.testAction(TriggerAutocompleteAction())` under the hood runs `CodyAutocompleteManager.instance.triggerAutocomplete(
        editor, offset, InlineCompletionTriggerKind.INVOKE)`.
All other calls of the `triggerAutocomplete` are with `InlineCompletionTriggerKind.AUTOMATIC`.
So you could simply check if `triggerKind == InlineCompletionTriggerKind.INVOKE` and it's something we are already doing in this 
class, e.g. to enchance logging verbosity.

Reply:
I'm going to address these issues in next PR.